Add issue template (#451)
thirdweb-dev/typescript-sdk: .github/ISSUE_TEMPLATE/bug_report.md

---
name: Bug report

> about: Create a report to help us improve

Reviewer: help us improve what?

title: ''

> labels: ''

Reviewer: can add `bug` here (since it's a bug report?)

> assignees: ''

Reviewer: can add @joaquim-verges and @ndeto here for now

---

**Prerequisites**

> * [ ] I've updated the `@thirdweb-dev/sdk` package to the latest version.

Reviewer: ```suggestion
* [ ] I have checked that `@thirdweb-dev/sdk` is on the latest _stable_ version and the bug still exists.
```

> * [ ] I've checked other issues and it's not duplicated, check the [issues list](https://github.com/thirdweb-dev/typescript-sdk/issues?q=is%3Aissue+is%3Aclosed).

Reviewer: ```suggestion
* [ ] I have checked [existing issues](https://github.com/thirdweb-dev/typescript-sdk/issues?q=is%3Aissue) and I was unable to find a duplicate of this.
```

**Optional**

* [ ] I am willing to contribute a PR that fixes the problem with some guidance.

**Describe the bug**
A clear and concise description of what the bug is.

**To Reproduce**
Steps to reproduce the behavior:
1. Install '...'
2. Initialize '....'
3. Execute the function '....'

> 4. See error

Reviewer: don't think we necessarily need this section

Reviewer: would prefer to get a reproduction repo

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Package versions (please complete the following information):**
 - `@thirdweb-dev/sdk`: [e.g. 2.2.6]
 - `node`: [e.g. v12.16.3]

>  - Frontend of your choice if applicable (`react`, `next`, `angular`, `vue`, etc.): [e.g. v12.16.3]

Reviewer: We really care about any build tools, etc here instead

Reviewer: as in, in terms of "fronend of your choice"

Reviewer: we want `sdk version` + `node version` + `any build tools, typescript vs not, etc`

> **Link to the project**

Reviewer: ```suggestion
**Reproduction link**
```

> If you have a reproducible project on [CodeSandbox](https://codesandbox.io/), [Replit](https://replit.com/), or [GitHub](https://github.com/), link it here. (This will make it much easier for us to reproduce your issue.)

Reviewer: ```suggestion
If you have a reproducible project on [CodeSandbox](https://codesandbox.io/), [Replit](https://replit.com/), or [GitHub](https://github.com/), link it here. (This will make it much easier for us to reproduce and fix your issue.)
```
